### PR TITLE
[enhancement] Added input timeout option to AntiAFK

### LINKED ISSUE #1215: AntiAfk timeout option
This would allow you to leave AntiAFK always on, and it would AntiAFK if you stop moving for a certain timeout, eg 60 seconds, but stop AntiAFK when either Baritone starts or you manually input something

### PR DESCRIPTION
**Pull request by Xiaro**
Closes #1215

### CODE REVIEW

**Comment by dominikaaaa** [comment]
Once #1220 is merged, I'll add this in a new pull, as AntiAfk was kotlinized

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt [hunk 1]
```diff
@@ -7,80 +7,138 @@ import me.zero.alpine.listener.EventHook
...
+        description = "Prevents being kicked for AFK"
+)
 class AntiAFK : Module() {
-    private val frequency = register(Settings.integerBuilder("ActionFrequency").withMinimum(1).withMaximum(100).withValue(40).build())
-    val autoReply = register(Settings.b("AutoReply", true))
-    private val swing = register(Settings.b("Swing", true))
-    private val jump = register(Settings.b("Jump", true))
+    private val delay = register(Settings.integerBuilder("ActionDelay").withValue(50).withRange(0, 100).build())
+    private val variation = register(Settings.integerBuilder("Variation").withValue(25).withRange(0, 50))
     private val squareWalk = register(Settings.b("SquareWalk", true))
```
Comment (dominikaaaa) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt:38]: Make this off by default, and rename it to "Walk" or something

Also, disabling it should cancel Baritone

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt [hunk 2]
```diff
@@ -7,80 +7,138 @@ import me.zero.alpine.listener.EventHook
...
+
+    override fun onEnable() {
+        with(BaritoneAPI.getSettings().disconnectOnArrival) {
+            baritoneDisconnectOnArrival = value
+            value = false
+        }
+    }
 
+    override fun onDisable() {
+        startPos = BlockPos(114514, -696969, 404)
```
Comment (dominikaaaa) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt:94]: Why are we doing this?

**Review by dominikaaaa — CHANGES_REQUESTED** [review]

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt [hunk 3]
```diff
@@ -7,80 +7,138 @@ import me.zero.alpine.listener.EventHook
...
 class AntiAFK : Module() {
-    private val frequency = register(Settings.integerBuilder("ActionFrequency").withMinimum(1).withMaximum(100).withValue(40).build())
-    val autoReply = register(Settings.b("AutoReply", true))
-    private val swing = register(Settings.b("Swing", true))
-    private val jump = register(Settings.b("Jump", true))
+    private val delay = register(Settings.integerBuilder("ActionDelay").withValue(50).withRange(0, 100).build())
+    private val variation = register(Settings.integerBuilder("Variation").withValue(25).withRange(0, 50))
     private val squareWalk = register(Settings.b("SquareWalk", true))
     private val radius = register(Settings.integerBuilder("Radius").withMinimum(1).withValue(64).build())
-    private val turn = register(Settings.b("Turn", true))
```
[the remark this answers is not in the document]
Comment (Xiaro) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt:32]: It got moved to companion object lol.

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt (hunk 2 above)
Comment (Xiaro) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt:94]: Just a dummy value telling that the blockpos is not initialized lol

**Review by l1ving — APPROVED** [review]

**Comment by l1ving** [comment]
Thanks!

**Pull request merged**